docs/getting-started.md

```diff
@@ -0,0 +1,44 @@
+# Getting started 
+
+## What is Luigi?
+
+Luigi is an open source JavaScript framework for micro frontends. It enables you to:
+* create a unified user interface or web application using **Luigi Core**. 
+* create micro frontends and connect them to the Luigi Core app using **Luigi Client**.
+
+## What are micro frontends? 
+
+Micro frontends are small frontend web applications that are implemented on one global user interface. They allow you to:
+
+* **break up big, monolithic UIs**: decompose your frontend into smaller chucks and make it more manageable, all while maintaining a consistent user experience.
+* **give individual teams autonomy**: every team in your project can develop their own end-to-end application which exists on the same core app. This gives teams full autonomy over release dates and updates, while also reducing code dependencies.
+* **extend functionality**: integrate external micro frontends to utilize functions from a 3rd party.
+* **scale**: add new features with ease without having to rely on only one UI team.
+* **make your website fast and resilient**: even if a micro frontend underperforms, the main user interface is not affected.
+* **be technology agnostic**: each micro frontend can be developed using a separate framework without having to synchronize everything with the main application. This enables you to quickly react to trends and adopt new technologies.
+
+## Prerequisites 
+
+All you need to get started with Luigi is basic knowledge of HTML and JavaScript. 
+
+You can build an application in Luigi without a framework, or use different frameworks such as:
+* Angular 6
+* SAPI5/OpenUI5
+* VUE.JS 
+
+## Next steps
+
+### Luigi Core
+
+To create a global user interface and host a full web application in Luigi:
+
+1. [Set up a Luigi Core application](https://github.com/SAP/luigi/blob/master/docs/application-setup.md).
+2. [Read the Luigi Core documentation](https://github.com/SAP/luigi/blob/master/docs/README.md#luigi-core).
+
+
+### Luigi Client
+
+To develop micro frontends and connect them to an already existing Luigi Core app:
```

Comment:
Same as above, maybe we can be more specific in the second point :)

Reply:
Thanks! I made these changes in the latest commit.